<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPainterClass</class>
 <widget class="QMainWindow" name="MyPainterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPainter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Nastavenia</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Graf</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Sínus</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kosínus</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tangens</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kotangens</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>polynom</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Typ grafu</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Bodový</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Čiarový</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>Stĺpcový</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Delenie</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="spinBox">
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>rizssah v rad</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>polynom</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Kresli</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Vymaž</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>484</width>
           <height>397</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEffects">
    <property name="title">
     <string>Effects</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEffects"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mypainter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MyPainterClass</receiver>
   <slot>ActionOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MyPainterClass</receiver>
   <slot>ActionSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>MyPainterClass</receiver>
   <slot>EffectClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MyPainterClass</receiver>
   <slot>ActionNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MyPainterClass</receiver>
   <slot>KresliClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MyPainterClass</receiver>
   <slot>EffectClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ActionOpen()</slot>
  <slot>ActionSave()</slot>
  <slot>EffectClear()</slot>
  <slot>ActionNew()</slot>
  <slot>KresliClicked()</slot>
 </slots>
</ui>
